x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/utils/MlIndexAndAliasTests.java

```diff
@@ -81,6 +81,7 @@
     private ArgumentCaptor<CreateIndexRequest> createRequestCaptor;
     private ArgumentCaptor<IndicesAliasesRequest> aliasesRequestCaptor;
 
+    @SuppressWarnings("unchecked")
```

Comment:
We can use the `anyActionListener()` trick here again, as well as defining a `mockActionListener()` for the other compiler error.

Reply:
Sure, we could do that, but like the other case, this requires defining 4 helpers plus address the casting on line 100, hence leaving this as is. But please lmk if you think otherwise.